CHANGELOG.md

```diff
@@ -33,6 +34,9 @@ BREAKING CHANGES:
   type: kubernetes.io/service-account-token
   ```
 
+* Control Plane
+  * Rename flag `server-address` to `token-server-address` in the `inject-connect` subcommand to avoid overloading the context of the `server-address` flag. [[GH-1426](https://github.com/hashicorp/consul-k8s/pull/1426)]
+
 IMPROVEMENTS:
```

Comment:
One last thing: could you add envoy version bump to changelog as an improvement?